### PR TITLE
don't log initalNode in standalone kubelets

### PR DESCRIPTION
**Pull request by mikedanese**
Logs are impressively spammy right now and not useful:

https://storage.googleapis.com/mikedanese/kubelet.log

/kind cleanup
/sig node

```release-note
NONE
```

### CODE REVIEW

**Comment by mikedanese** [comment]
hmm....
/shrug
/retest

File: pkg/kubelet/kubelet_getters.go [hunk 1]
```diff
@@ -228,12 +228,10 @@ func (kl *Kubelet) GetNode() (*v1.Node, error) {
...
 func (kl *Kubelet) getNodeAnyWay() (*v1.Node, error) {
-	if kl.kubeClient != nil {
-		if n, err := kl.nodeInfo.GetNodeInfo(string(kl.nodeName)); err == nil {
-			return n, nil
-		}
+	if kl.kubeClient == nil {
+		return kl.initialNode(false)
 	}
-	return kl.initialNode()
+	return kl.nodeInfo.GetNodeInfo(string(kl.nodeName))
```
Comment (tallclair) [inline pkg/kubelet/kubelet_getters.go]: This is changing the logic, since it won't fall back to initialNode in case of an error.

Comment (mikedanese) [reply pkg/kubelet/kubelet_getters.go]: Oops. Nice catch.

**Comment by mtaufen** [comment]
Agreed that's pretty verbose.

Instead of the bool arg, I think it would be cleaner to just increase the
required log level and print the json for the returned node in the one
place you actually want it to log by default.

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

**Pull request closed without merging**